lsp-mode.el

```diff
@@ -2722,33 +2722,34 @@ in that particular folder."
 (defun lsp--send-open-close-p ()
   "Return whether open and close notifications should be sent to the server."
   (let ((sync (gethash "textDocumentSync" (lsp--server-capabilities))))
-    (and (hash-table-p sync)
-         (gethash "openClose" sync))))
+    (and (or (memq sync '(1 2))
```

Comment:
It will work as it is but I think that the outside `and` is not needed(it has only one argument).

Reply:
Just removed it 👍